_includes/1.0/overview/security/oauth2/downloads.html

```diff
@@ -0,0 +1,29 @@
+<div id="downloads" class="group">
+
+    <h2>Support Downloads for Development</h2>
+
+    <p>The following downloads can be used by developers to validate the client against OneUp <code>staging</code> environment</p>
+
+    <p>
+        <form method="get" action="{{ site.url }}/downloads/http-client.zip">
```

Comment:
I think you reversed the link between IntelliJ and Postman collection.

Reply:
Fixed with the commit [908f6b3724d13700528479bf0e305807c8ca9161](https://github.com/myERP/myerp.github.io/pull/31/commits/908f6b3724d13700528479bf0e305807c8ca9161)